lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1051,10 +1051,10 @@ def test_all_permutations(self):
 @tests.skip_data
 class Test_slices_over(tests.IrisTest):
```

Comment:
There are several comments mentioning `420` and `100x100` that also need updating.